modular/signer/signer_client.go

```diff
@@ -98,17 +101,32 @@ func NewGreenfieldChainSignClient(rpcAddr, chainID string, gasLimit uint64, oper
 		return nil, err
 	}
 
+	gcKM, err := keys.NewPrivateKeyManager(gcPrivateKey)
+	if err != nil {
+		return nil, err
+	}
+	gcClient, err := client.NewGreenfieldClient(rpcAddr, chainID, client.WithKeyManager(gcKM))
+	if err != nil {
+		return nil, err
+	}
+	gcAccNonce, err := gcClient.GetNonce()
+	if err != nil {
+		return nil, err
+	}
+
 	greenfieldClients := map[SignType]*client.GreenfieldClient{
 		SignOperator: operatorClient,
 		SignFunding:  fundingClient,
 		SignSeal:     sealClient,
 		SignApproval: approvalClient,
+		SignGc:       gcClient,
 	}
 
 	return &GreenfieldChainSignClient{
 		gasLimit:          gasLimit,
 		greenfieldClients: greenfieldClients,
 		sealAccNonce:      sealAccNonce - 1,
+		gcAccNonce:        gcAccNonce,
```

Comment:
The inited value is `gcAccNonce -1` in develop branch, and what is the difference?

Reply:
let me review this part